kqueen/auth/common.py

```diff
@@ -12,18 +13,36 @@
 logger = logging.getLogger('kqueen_api')
 
 
```

Comment:
you can set  
AUTH_MAP =
 {'ldap' = {"engine": "LDAPAuth",
+            "param": {
+                "uri": config.get('LDAP_URI')
+            }
+            }
+    'local' = {"engine": "LocalAuth",
+             "param": {}
+             }
}

here and not to create a new class